depthai_sdk/src/depthai_sdk/oak_camera.py

```diff
@@ -175,7 +175,8 @@ def create_stereo(self,
                       left: Union[None, dai.Node.Output, CameraComponent] = None,  # Left mono camera
                       right: Union[None, dai.Node.Output, CameraComponent] = None,  # Right mono camera
                       name: Optional[str] = None,
-                      encode: Union[None, str, bool, dai.VideoEncoderProperties.Profile] = None
+                      encode: Union[None, str, bool, dai.VideoEncoderProperties.Profile] = None,
+                      colormap: dai.Colormap = None
```

Comment:
What if we expose the colormap setting via other api, not on init?

Reply:
Hm, we also expose it in `StereoComponent.config_stereo`, what do you think?